api/v1/scopeinstance_types.go

```diff
@@ -37,8 +37,21 @@ type ScopeInstanceSpec struct {
 type ScopeInstanceStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// Conditions represent the latest available observations of an object's state
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
 }
 
+// TODO(everettraven): Add Condition Types and Reasons as part of the API
+const (
+	ScopeInstanceSucceededType = "Succeeded"
+
+	ScopeTemplateNotFoundReason         = "ScopeTemplateNotFound"
+	RoleBindingDeleteFailureReason      = "RoleBindingDeleteFailure"
+	RoleBindingCreateFailureReason      = "RoleBindingCreateFailure"
+	ScopeInstanceReconcileSuccessReason = "ScopeInstanceReconcileSuccess"
+)
```

Comment:
@joelanford does this resemble what you meant by making the constants part of the API? Should all the reasons/types be unique to a certain API type or if there are common ones should they be moved to a more common location?

Reply:
TODO @everettraven : update this to be more like the rukpak implementation

Reply:
Updated in commit https://github.com/awgreene/oria-operator/pull/21/commits/01d4b49156e5523bab706fd1c4c06ee6d1fa057c